openmc/model/model.py

```diff
@@ -259,13 +261,6 @@ def from_model_xml(cls, path='model.xml'):
         model.materials = openmc.Materials.from_xml_element(root.find('materials'))
         model.geometry = openmc.Geometry.from_xml_element(root.find('geometry'), model.materials)
 
-        # gather meshes from other classes before reading the tally node
-        if model.settings.entropy_mesh is not None:
-            meshes[model.settings.entropy_mesh.id] = model.settings.entropy_mesh
-
-        for ww in model.settings.weight_windows:
-            meshes[ww.mesh.id] = ww.mesh
-
```

Comment:
Correct me if I'm wrong, but `meshes` is already updated from the call to `Settings.from_xml_element(...)` so this is not needed at all

Reply:
You're correct. These are no longer needed.